scylla-server/src/services/systems.services.ts

```diff
@@ -0,0 +1,15 @@
+import prisma from '../prisma/prisma-client';
+import { ResponseFunction } from '../utils/message-maps.utils';
+
+// CRUD operation to get all systems with ResponseFunction type
+
```

Comment:
use JavaDoc i.e /** */ it should prompt you with paramter values when you do the autocomplete with VSCode